introduction.tex

```diff
@@ -20,22 +20,71 @@ \chapter{Introduction}
 
 \section{Terminology}
 
-A \emph{platform} is a single integrated circuit consisting of one or more
-\emph{components}. Some components may be RISC-V cores, while others may have a different
-function. Typically they will all be connected to a single system bus.
-A single RISC-V core contains one or more hardware threads, called
-\emph{harts}.
-
-\emph{DXLEN} of a hart is its widest supported XLEN, ignoring the current value
-of \Fmxl in \Rmisa.
-
-A \emph{physical address} is directly usable on the system bus.
-
-A \emph{virtual address} is an address as a hart sees it. If there is address
-translation this may be different from the \emph{physical address}. If there is
-no translation then it will be the same.
-
-\subsection{Context}
+\begin{description}[style=nextline]
+    \item[AMO]
+        Atomic Memory Operation.
+    \item[BIST]
+        Built-In Self Test.
```

Comment:
BIST is only used in one place in the document so it's probably easier to just say "BIST (built-in self test)" in that one location instead of having the reader refer to the glossary.

Reply:
Done.